From: Giovanni Campagna <gcampagna@src.gnome.org>
Date: Thu, 31 Jan 2013 12:43:50 +0000
Subject: Remove a deprecated key

This key has been around for little enough that it is probably safe
to remove, and removing it reduces confusion in users and developers.

https://bugzilla.gnome.org/show_bug.cgi?id=692944
---
 schemas/org.gnome.desktop.screensaver.gschema.xml.in | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/schemas/org.gnome.desktop.screensaver.gschema.xml.in b/schemas/org.gnome.desktop.screensaver.gschema.xml.in
index 5c2db2a..b43377e 100644
--- a/schemas/org.gnome.desktop.screensaver.gschema.xml.in
+++ b/schemas/org.gnome.desktop.screensaver.gschema.xml.in
@@ -21,6 +21,16 @@
       <summary>Time before locking</summary>
       <description>The number of seconds after screensaver activation before locking the screen.</description>
     </key>
+    <key name="show-notifications" type="b">
+      <default>false</default>
+      <summary>Show notifications in the lock screen</summary>
+      <description>
+        Whether notifications are shown in the lock screen or not. This only affects the standard experience.
+
+        DEPRECATED: This key is deprecated and ignored.
+        It has been superseded by org.gnome.desktop.notifications show-in-lock-screen.
+      </description>
+    </key>
     <key name="show-full-name-in-top-bar" type="b">
       <default>true</default>
       <summary>Show full name in the lock screen</summary>
